core/src/main/resources/com/squarespace/template/plugins/platform/variants-select.html

```diff
@@ -12,7 +12,8 @@
         <option value="">{selectText|message variantName:name}</option>
         {.repeated section values}<option value="{@|htmltag}">{@|htmltag}</option>{.end}
       </select>
-    </div>
+    </div>{.repeated section values}
+    <div class="variant-box" value="{@|htmltag}">{@|htmltag}</div>{.end}
```

Comment:
We should use radio inputs here instead of just divs for a11y support. See the markup for the Haus as an example: https://drink.haus/collections/shop. Also is there a reason for the indentation? Could we move the {.repeated} part down onto its own line?

![Screen Shot 2020-04-21 at 3 16 56 PM](https://user-images.githubusercontent.com/1840976/79904991-8f1bcc80-83e3-11ea-9026-9ce6d1613c90.png)

Reply:
Oh neat, will definitely change to radio inputs then. The indentation on the repeated tag is because template-compiler will strip the repeated tag when rendering markup, which turns into additional whitespace due to the new line, i.e.:

```
<div class="foo" />
{.repeated section values}
<div class="bar" />
{.end}
```

will become 

```
<div class="foo" />
 <div class="bar" />
```